settings.html

```diff
@@ -91,6 +98,21 @@ <h2 class="settings-field__title" data-i18n="settings.bg.title">Blood sugar leve
             <label class="settings-input__label" for="bg-low" data-i18n="settings.bg.low">Low level threshold</label>
           </div>
         </section>
+        <section class="settings-field settings-fields__other">
```

Comment:
The `.settings-input` is a wrapper for one input element of the form. If you want to put more than one element, you should wrap it in its wrapper to properly set the CSS for them in the grid.